internal/loader/fsloader.go

```diff
@@ -38,6 +38,7 @@ const (
 // LoadTrees loads several paths, wrapping them all in virtual folder.
```

Comment:
i just realized this seems to have some secret default behaviour.  I guess it's ok to keep that for the pros - but we should warn since we're disablin it at the cmd line level